docs/capabilities.md

```diff
@@ -12,7 +12,7 @@ Capability&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbs
 `appium:platformVersion` | The platform version of an emulator or a real device. This capability is used for device autodetection if `udid` is not provided
 `appium:udid` | UDID of the device to be tested. Could be retrieved from Xcode->Window->Devices and Simulators window. Always set this capability if you run parallel tests or use a real device to run your tests.
 `appium:noReset` | Prevents the device to be reset before the session startup if set to `true`. This means that the application under test is not going to be terminated neither its data cleaned. `false` by default
-`appium:fullReset` | Being set to `true` always enforces the application under test to be fully uninstalled before starting a new session. `false` by default
+`appium:fullReset` | Being set to `true` always enforces the application under test to be fully uninstalled before starting a new session. If the application under test remains on the device as an [offload application](https://discussions.apple.com/thread/254887240), the application data will remain. Please use [`mobile: removeApp`](execute-methods.md#mobile-removeapp) to ensure the application data removal. `false` by default
```

Comment:
- not sure if this comment makes sence for simulators as we perform the reset for the whole device
- for real devices it would be better to just add a link to a troubleshooting article